### ISSUE TITLE
[BUG] Missing SASL Config Due to String Casing

### ISSUE THREAD

**Issue by nickpoulos** [mateusjunges/laravel-kafka#69]
**Describe the bug**
I was trying to debug why my SASL configs were not being loaded.   Everything looked correct, except the `usingSasl()` function below was still returning false.  This was because it checks specifically for uppercase strings, whereas our config was in  lowercase. 

```php
private function usingSasl(): bool
{
    return $this->securityProtocol === 'SASL_PLAINTEXT' || $this->securityProtocol === 'SASL_SSL';
}
```

Because Kafka itself seems to understand the lowercase `sasl_plaintext` values, I suggest we change the function to ignore case by throwing a `strtolower` in there. 

**To Reproduce**
1. Setup SASL, but use lowercase `sasl_plaintext` as the security protocol. 
2. See that SASL configs are not loaded via `getConsumerOptions()`

**Expected behavior**
I would expect the SASL configs to be loaded properly regardless of the casing of their values.

**Comment by rtuin** [comment]
Makes sense to me!

**Comment by mateusjunges** [comment]
@nickpoulos @rtuin released on v1.6.1. I'll add this to v1.4.x and v1.5.x later today.
